src/Controls/samples/Controls.Sample.UITests/Issues/Issue5555.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Maui.Controls;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.None, 5555, "Memory leak when SwitchCell or EntryCell", PlatformAffected.iOS)]
+	public class Issue5555 : TestContentPage
```

Comment:
Could with the changes from this PR include the ListView in the `HandlerDoesNotLeak `test?

Reply:
Yea, this scenario is somewhat specific to being pushed to and then having a layout performed with the app. I'd worry about being inaccurate moving to `HandlerDoesNotLeak`